packages/components/Button/src/utils.android.ts

```diff
@@ -0,0 +1,48 @@
+import { memoize } from '@fluentui-react-native/framework';
```

Comment:
Could we rename this file to `extractStyle.ts` or the like, and then instead just have a comment explaining this is used for Android's ripple container?